src/pages/playertools.tsx

```diff
@@ -33,8 +83,9 @@ const PlayerToolsPage = () => {
 	const [eventData, setEventData] = useStateWithStorage('tools/eventData', undefined);
 	const [activeCrew, setActiveCrew] = useStateWithStorage('tools/activeCrew', undefined);
 
+	const urlParams = new URLSearchParams(window.location.search);
```

Comment:
[website.e0ab816a-7a0d-494f-9ef4-2605d4872e96.log](https://github.com/stt-datacore/website/files/7143560/website.e0ab816a-7a0d-494f-9ef4-2605d4872e96.log)
This is causing a build failure here.